sql/postgresql/README.md

```diff
@@ -25,7 +25,73 @@ createdb my_pap_db
 psql --dbname=my_pap_db --file="<pingauthorize-contrib-root>/sql/postgresql/init-policy-database.sql" >/dev/null
 ```
 You should now have an initialized policy database. For instructions on configuring the Policy Editor to connect to
-your PostgreSQL instance, see "[Installing the PingAuthorize Policy Editor manually](https://docs.pingidentity.com/csh?Product=paz-latest&context=paz_install_pe_noninteractive)".
+your PostgreSQL instance, see [Installing the PingAuthorize Policy Editor manually](https://docs.pingidentity.com/csh?Product=paz-latest&context=paz_install_pe_noninteractive).
+
+
+### Upgrading an existing policy database to a newer schema
+1. Verify the current version of your policy database using one of the following methods.
+
+**Optional** - Access `my_pap_db` and query the `TAG` column of `my_pap_db.databasechangelog`
+
+```
+psql -U postgres -d my_pap_db
+```
+
+```
+SELECT TAG FROM DATABASECHANGELOG WHERE TAG IS NOT NULL ORDER BY TAG DESC LIMIT 1;
+```
+
+**Optional** - Follow the [steps to non-interactively set up](https://docs.pingidentity.com/csh?Product=paz-latest&context=paz_install_pe_noninteractive)
+   the target version of the Policy Editor. If your schema is out of date, you will be prompted to run the intermediate upgrade scripts sequentially.
```

Comment:
I think it would be worthwhile to add a warning to clarify that `setup` is intended to be run against a clean install of the Policy Editor. The example command below would not work as-is if there are existing `configuration.yml` or certificate keystores/truststores.  It's not the best user experience, but currently this approach sets up a separate Policy Editor install (as opposed to an in-place upgrade) for the sole purpose of identifying the upgrade scripts.

Can you add some clarifying content explaining that during `start-server`, the application checks if the database schema requires upgrades, and if so, prints the locations of the upgrade scripts, and that the `setup` command below is provided if the customer wishes to avoid affecting their existing installation files?